pyproject.toml

```diff
@@ -2,11 +2,92 @@
 requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "21cmSense"
+description = "Compute sensitivies of radio interferometers to the cosmic 21cm signal."
+readme = "README.rst"
+authors = [
+  {name="Jonathan Pober", email="jonathon_pober@brown.edu"},
+  {name="Steven Murray", email="murray.steveng@gmail.com"},
+  {name="Matt Kolopanis", email="matthew.kolopanis@asu.edu"},
```

Comment:
actually matthew.kolopanis@gmail.com would be better here for me please.

Reply:
done! And what good luck that you get to have your name @ gmail without numbers or weird gymnastics!

Reply:
the advantages of having a fairly unique name.